keycloak-export/src/main/resources/theme/ctexport/admin/resources/js/services.js

```diff
@@ -5,14 +5,20 @@ var module = angular.module('keycloak.services', [ 'ngResource', 'ngRoute' ]);
 module.service('Dialog', function($modal) {
     var dialog = {};
 
-    var openDialog = function(title, message, btns, template) {
+    var openDialog = function(title, message, btns, template, sensitive = false, name) {
```

Comment:
sensitive=false: is it a default value? Doesn't name should have a default value? (I guess this function can be called from already existing parts of the code)

Reply:
- yeah it's a default value so that I don't have to update the many many calls to openDialog all over the code
- when "sensitive" is false, "name" is not used, so "undefined" is ok.